### DISCUSSION TITLE
Utility React Hooks

### DISCUSSION THREAD

**Discussion by coopbri** [chakra-ui/ark#817] [Q&A] [answered]
After trying Ark and Panda I am super excited about the distributed future of the Chakra ecosystem. One question I have that was unclear to me from [Segun's article](https://www.adebayosegun.com/blog/the-future-of-chakra-ui#how-does-ark-fit-in-chakra-ui) and the docs was the place of some of the utility React hooks Chakra has, such as `useDisclosure` and `useBreakpointValue`. Will these be present in Ark (or Zag), or is the projection that these will continue to be an exclusive part of Chakra itself?

I did see Ark's website is using [`usehooks-ts`](https://usehooks-ts.com) (https://github.com/chakra-ui/ark/pull/312).

**Comment by cschroeter** [comment, marked as answer]
We have not decided how to handle this. At the moment we are going to focus on components and setting up the ecosystem. `usehooks-ts` is defintively a great library to use for now.
